ApiBundle/Controller/NodeController.php

```diff
@@ -33,13 +33,16 @@ class NodeController extends BaseController
      */
     public function showAction(Request $request, $nodeId)
     {
-        $language = $request->get('language');
+        $currentSiteManager = $this->get('open_orchestra.manager.current_site');
+        $language = (null !== $request->get('language')) ? $request->get('language') : $currentSiteManager->getCurrentSiteDefaultLanguage();
```

Comment:
you use the @nicolasThal syntax : $request->get('language', $currentSiteManager->getCurrentSiteDefaultLanguage())